pytest.ini

```diff
@@ -6,7 +6,7 @@ markers =
     integration: mark an integration test
     skip_master: mark a test that is temporarily skipped for mxnet master validation.
 
-timeout = 30
+timeout = 60
```

Comment:
Why do we double the timeout?

Reply:
Because each tests gets a bit slower when executing things in parallel.